src/js/view/controls/components/timeslider.js

```diff
@@ -109,8 +109,16 @@ class TimeSlider extends Slider {
         // Show the tooltip on while dragging (touch) moving(mouse), or moving over(mouse)
         this.elementUI = new UI(this.el, { useHover: true, useMove: true })
             .on('drag move over', this.showTimeTooltip.bind(this), this)
-            .on('dragEnd out', this.hideTimeTooltip.bind(this), this)
+            .on('out', this.hideTimeTooltip.bind(this), this)
```

Comment:
You shouldn't need `bind(this)` when passing `this` as the listener context (third param) for `on`:

`.on('event', fn.bind(this))` is the same as `.on('event', fn, this)` and it's best to avoid `bind` since it can be replaced on `Function.prototype`.